README.md

```diff
@@ -12,13 +12,16 @@ go-bitswap
 
 ## Table of Contents
 
+- [Background](#background)
 - [Install](#install)
-- [Protocol](#protocol)
+- [Usage](#usage)
 - [Implementation](#implementation)
 - [Contribute](#contribute)
 - [License](#license)
 
-## Protocol
+
+## Background
+
```

Comment:
> Bitswap is a message based protocol, as opposed to response-reply

Not part of this PR but should this be `Bitswap is a message based protocol, as opposed to request-response`?